### PR TITLE
Minor clean up.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: docs/Architecture.md [hunk 1]
```diff
@@ -1 +1 @@
...

**[Aggregate Service](Runbook.md#aggregate-service)**. This service provides the back-end API for reports against the OLAP data store, i.e. aggregate reports. It is horizontally scalable; however, the OLAP data store is typically the bottleneck and running many instances of this service may not result in better overall throughput.

**[Admin Service](Runbook.md#admin-service)**. This service provides the back-end API for administrative functionality including management of student groups, instructional resource links, embargo settings. It is horizontally scalable, however administrative management is limited in scope and a single instance is usually sufficient.

**[Report Processor](Runbook.md#report-processor)**. This processor generates PDF reports. It is horizontally scalable and many instances should be run to deal with reporting load.  

**[PDF Generator](Runbook.md#pdf-generator)**. This application converts HTML to PDF. It is used by the report processor. It is horizontally scalable and many instances should be run to deal with reporting load.
```
Comment (agorina) [inline docs/Architecture.md]: The only change is "subject definitions", you will see of you search for it. I do not know why it shows that much difference.

Comment (jtreuting) [inline docs/Architecture.md]: any way to fix the line breaks here?

File: docs/Troubleshooting.md [hunk 2]
```diff
@@ -170,6 +170,8 @@ It looks like a number of schools are parented by a state. You can see the paylo
 <a name="migrate"></a>
 #### Migrate
 
+There are two migrate services: Migrate Reporting and Migrate OLAP. The below instructions have samples for the Migrate Reporting. They could be applied to Migrate OLAP by replacing the service and/or database schema names to reference the appropriate instances.
```
Comment (jtreuting) [inline docs/Troubleshooting.md]: above you called out the specific things to change.  might be worth doing that here as well?

File: docs/Architecture.md (hunk 1 above)
Comment (agorina) [reply docs/Architecture.md]: I could not find an easy one. Do you know of any?

**Pull request merged**